helm/sidecarinjector/values.yaml

```diff
@@ -33,6 +33,10 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+sidecarconfig: 
+
+mutationconfig: 
```

Comment:
what is this needed for ?

Reply:
good catch, thx... initially i planned to pass those as helm values but `kubectl apply -f configmap` seems simpler.